### ISSUE TITLE
rom placed on "sd card root" does not work

### ISSUE THREAD

**Issue by goofyseeker311** [MiSTer-devel/Minimig-AGA_MiSTer#172]
so where the rom should be placed, /media/fat/ and ./games/amiga/ does not work.
with the amiga forever 10 roms 1.3/3.1 renamed to kick.rom or KICK.ROM with recommended settings.
even selecting the rom location from the settings does not make the rom loading work.

**Comment by sorgelig** [comment]
in /media/fat/Amiga/
after selecting, save config to default.

**Comment by goofyseeker311** [comment]
did not work before. have you tried recently. I cant guarantee amiga forever 10 roms files are correct tho.

not /media/far/games/amiga? but /media/fat/amiga? nope none of the options for the rom location work.

maybe amiga forever 10 roms are broken?

**Comment by sorgelig** [comment]
ah, sorry /media/far/games/amiga of course.
not sure about amigaforever roms. Aren't they encrypted? Minimig supports some encrypted roms, but i'm not sure which exactly.

**Comment by goofyseeker311** [comment]
yeah they were, had to use the rom toolbox to get the normal roms.

one button would make things easier, swap disks button (between loaded disks).
edit: alien breed 1992 special edition does not want to continue from "insert disk 2" when loading even if the correct working disk (winuae works with 1.3 kickstart rom) is inserted. not with mouse clicks or fire button press. 

edit: with 1.3 kickstart rom a500 the fire button does not work when it asks for disk change but with kickstart 3.1 rom a1200 the fire button works and the game loads fully.

edit: numlock as (undocumented) feature to change between keyboard/mouse/joystick1/joystick2 modes. ok got it.

edit: kick 3.1 a1200 sounds are hilariously off (also using winuae) for alien breed 1992 special edition. better use 1.3 rom on a500.

**Comment by sorgelig** [comment]
There is a project WHDload making many games load easier with compatibility patches applied. So you may have a whole game collection of HDD without messing with different kickstarts and HW configs.

**Comment by goofyseeker311** [comment]
cool cool. so why is it not included in the core like its in amiga forever winuae.

**Comment by sorgelig** [comment]
Because WHDLoad is Amiga software. Repository doesn't deal with software. Only Core specific drivers/addons are included. You are free to choose which apps to use in Amiga.

**Comment by limi** [comment]
> cool cool. so why is it not included in the core like its in amiga forever winuae.

Check out the https://amiga.vision project (disclaimer: I am the maintainer)

**Comment by goofyseeker311** [comment]
most stuff work now, but minimig not handling multiple disk games gracefully is not yet happening.

for example, a two disk game xenon 2, asks "insert game disk" even disk 2 is in the df1 inserted, and disk 1 is in the df0 inserted.
only starts working if the disk 2 is swapped into df0 instead of disk 1. I think multiple disk dfx search is the default amiga behaviour.
same disk asking behaviour for both kickstart rom 1.2 a500 ocs and 3.1 a1200 aga.

I think it is not up to games to search from multiple drives but its the os rom function internally or anything between like the minimig emulator. so the os rom 1.2 and 3.1 does not see the drives as it should, to be able to find the second disc automatically.

no whdl software or anything. just as if there was a second disk drive as hardware in the normal a500/a1200 system.

**Comment by goofyseeker311** [comment]
so if you place the first disk 1 in the df1 slot, and the game disk 2 in the df0 slot, ie swapped non-logically, then it works, automatically, correctly. os rom system issue, or minimig issue? are the disk drives df0 and df1 not exactly equal.

edit: seems like the df1 game disk 1, df0 game disk 2 works only in 3.1 rom not in 1.2 rom. I would guess that any multiple disk drive detection is only on some later amiga os roms.

**Comment by goofyseeker311** [comment]
another issue with multiple fire buttons: running dynamite dux on a500-1.2-ocs (a1200-3.1-aga has graphical issues) does not seem to do the jump action even if minimig buttons are assigned to more than one fire button. the second and more fire buttons does not seem to be self-explanatory in mister/minimig case. wonder what is the source of the issue, the game should not be simplifying it to just one button tho. so suspecting mister/minimig emulation issues.
